{
  "comments": [
    {
      "key": {
        "uuid": "Sk15XmQU",
        "filename": "user/test/com/google/gwt/user/client/rpc/LoggingRPCTest.java",
        "patchSetId": 1
      },
      "lineNbr": 125,
      "author": {
        "id": 5010
      },
      "writtenOn": "2013-03-27T21:19:08Z",
      "side": 1,
      "message": "How about calling reportUncaughtException()? It will go there anyway but calling it directly means we don\u0027t have to wrap the exception. (Similarly below.)",
      "revId": "d5abd3332caa5e510ceabee68a49f0e0ba125bea",
      "serverId": "84fdd5b9-d3fc-3149-9ae9-e619898c4c46",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "Sk0S2rOU",
        "filename": "user/test/com/google/gwt/user/client/rpc/LoggingRPCTest.java",
        "patchSetId": 1
      },
      "lineNbr": 125,
      "author": {
        "id": 5110
      },
      "writtenOn": "2013-03-28T22:00:51Z",
      "side": 1,
      "message": "I agree but I don\u0027t want to single this one out as this is the pattern in all rpc tests.",
      "parentUuid": "Sk15XmQU",
      "revId": "d5abd3332caa5e510ceabee68a49f0e0ba125bea",
      "serverId": "84fdd5b9-d3fc-3149-9ae9-e619898c4c46",
      "unresolved": false
    }
  ]
}